src/app/debug/table/table.component.html

```diff
@@ -55,6 +55,10 @@
 
     </div>
 
+    @if (hasTimedOut) {
+      <h4 class="progress-report-timout">[One or more reports are in progress for more than {{ (this.reportsInProgressThreshold / 1000 / 60) }} minutes]</h4>
+    }
+
     <app-button [text]="'Compare'" [title]="'Compare'" *ngIf="showCompareButton()" (click)="compareTwoReports()"></app-button>
     <app-button [text]="'Open In New Tab'" *ngIf="showOpenInTabButton()" (click)="openReportInTab()"></app-button>
```

Comment:
No binding needed

Reply:
This means you can change it to just use interpolation since you're not really assigning a variable
```
<app-button [text]="'Compare'" [title]="'Compare'" ...></app-button>
```
Should become
```
<app-button text="Compare" title= "Compare" ...></app-button>
```